### PR TITLE
examples: add podman based AD joined winbind+smbd pod

### PR DESCRIPTION
**Pull request by phlogistonjohn**
Adds a script, config json template, and readme file to give a brief
into to running samba containers as a domain member.

### CODE REVIEW

File: examples/podman/README.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
+
+# Podman Examples
+
+These examples demonstrate using samba container images with podman.
+
+## Domain Joined SMB Shares
+
+Included here is a shell script and a JSON config template that
+demonstrates using podman to run an smbd instance joined to Active Directory.
```
Comment (obnoxxx) [inline examples/podman/README.md:9]: I think we might make it a little more clear that this examples requires an existing AD domain that the winbindd container will actually join with the credentials provided in the script.

**Review by obnoxxx — CHANGES_REQUESTED** [review]
Great stuff!
Just one comment inline (can we make it more clear that we're joining an *existing* domain?)

File: examples/podman/README.md (hunk 1 above)
Comment (phlogistonjohn) [reply examples/podman/README.md:9]: Sure. Will do.

Comment (phlogistonjohn) [reply examples/podman/README.md]: Done.

**Comment by phlogistonjohn** [comment]
Added a section to the doc touching on the potential need to configure /etc/sub{uid,gid} files if running this with rootless podman.

**Review by obnoxxx — APPROVED** [review]
/lgtm

thanks!

**Pull request merged**